src/rfac/rfr1.c

```
@@ -0,0 +1,81 @@
+ * common use. Instead of \f$ S |I_t| \f$, \f$ S|\frac{I_t - S*I_t}{\delta E}|
+ * \f$ is used.
+ *
+ * \param[in] eng pointer to list of energy values.
+ * \param[in] e_int pointer to list of experimental intensity values.
+ * \param[in] t_int pointer to list of theoretical intensities.
+ *
+ * \note \p eng must be terminated by #F_END_OF_LIST.
+ *
+ * \return R1-factor if successful, F_FAIL if failed.
+ */
+real rfac_r1(const real *eng, const real *e_int, const real *t_int)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rfac_r1' of similar type ('const int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)